packages/cozy-client/src/associations/helpers.js

```diff
@@ -1,33 +1,60 @@
+import pick from 'lodash/pick'
+import pickBy from 'lodash/pickBy'
 import Association from './Association'
+import HasOneInPlace from './HasOneInPlace'
+import HasMany from './HasMany'
+import HasManyInPlace from './HasManyInPlace'
+import HasManyFiles from './HasManyFiles'
 
-export const dehydrateDoc = document => {
-  const dehydrated = {}
-  Object.entries(document).forEach(([key, value]) => {
-    if (!(value instanceof Association)) {
-      dehydrated[key] = value
-    } else if (value.raw) {
-      dehydrated[key] = value.raw
-    }
+export const pickTypeAndId = x => pick(x, '_type', '_id')
+const applyHelper = (fn, objOrArr) =>
+  Array.isArray(objOrArr) ? objOrArr.map(fn) : fn(objOrArr)
+
+export const responseToRelationship = response =>
+  pickBy({
+    data: applyHelper(pickTypeAndId, response.data),
+    meta: response.meta,
+    next: response.next,
+    skip: response.skip
   })
-  return dehydrated
+
+const aliases = {
+  'io.cozy.files:has-many': HasManyFiles,
+  'has-many': HasMany,
+  'belongs-to-in-place': HasOneInPlace,
```

Comment:
I this just a convenience alias for `has-one-in-place` or am I missing something?